cmd/auth-rest/startcmd/start.go

```diff
@@ -440,6 +443,11 @@ func startAuthService(parameters *authRestParameters, srv server) error {
 		router.HandleFunc(handler.Path(), handler.Handle()).Methods(handler.Method())
 	}
 
+	interact, err := redirect.New()
```

Comment:
We should add a TODO as AS need to support different flows.

Reply:
TODO added